.github/workflows/build.yml

```diff
@@ -35,15 +34,15 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node }}
-          registry-url: 'https://registry.npmjs.org'
+          registry-url: "https://registry.npmjs.org"
 
       - name: Use Python 3.11
         uses: actions/setup-python@v4
         with:
-          python-version: '3.11'
+          python-version: "3.11"
 
-      - name: Build and package (Windows, Linux)
-        if: runner.os == 'Windows' || runner.os == 'Linux'
+      - name: Build and dev package (Windows, Linux)
```

Comment:
Nit: I'd probably rename the steps to Build dev package and Build prod package now.